app/javascript/components/admin/roles/Roles.jsx

```diff
@@ -14,7 +14,7 @@ import { useAuth } from '../../../contexts/auth/AuthProvider';
 export default function Roles() {
   const { t } = useTranslation();
   const [searchInput, setSearchInput] = useState();
-  const { data: roles, isLoading } = useRoles(searchInput);
+  const { data: roles, isLoading } = useRoles({ search: searchInput });
```

Comment:
Approaching a more of a named args style for `useRoles` to improve verbosity,
More on this below,